pgrx-pg-sys/build.rs

```diff
@@ -116,20 +114,15 @@ fn main() -> eyre::Result<()> {
 
     eprintln!("build_paths={build_paths:?}");
 
-    let pgrx = Pgrx::from_config()?;
-
-    println!("cargo:rerun-if-changed={}", Pgrx::config_toml()?.display().to_string(),);
-    println!("cargo:rerun-if-changed=include");
-    println!("cargo:rerun-if-changed=cshim");
-    emit_missing_rerun_if_env_changed();
+    emit_rerun_if_changed();
 
-    let pg_configs: Vec<(u16, PgConfig)> = if std::env::var(
+    let pg_configs: Vec<(u16, PgConfig)> = if env_tracked(
         "PGRX_PG_SYS_GENERATE_BINDINGS_FOR_RELEASE",
     )
     .unwrap_or("false".into())
         == "1"
```

Comment:
can't this be more elegantly stated as
```suggestion
    let pg_configs: Vec<(u16, PgConfig)> = if env_tracked(
        "PGRX_PG_SYS_GENERATE_BINDINGS_FOR_RELEASE",
    ).as_ref() == Some("1")
```